extensions/vertx/deployment/src/main/java/io/quarkus/vertx/core/deployment/VertxCoreProcessor.java

```diff
@@ -236,11 +234,10 @@ CoreVertxBuildItem build(
             BuildProducer<EventLoopSupplierBuildItem> eventLoops,
             ExecutorBuildItem executorBuildItem) {
 
-        Collections.sort(vertxOptionsConsumers);
-        List<Consumer<VertxOptions>> consumers = new ArrayList<>(vertxOptionsConsumers.size());
-        for (VertxOptionsConsumerBuildItem x : vertxOptionsConsumers) {
-            consumers.add(x.getConsumer());
-        }
+        List<Consumer<VertxOptions>> consumers = vertxOptionsConsumers.stream()
+                .sorted()
+                .map(cbi -> cbi.getConsumer())
+                .toList();
```

Comment:
This change is only there to avoid modifying the list of `vertxOptionsConsumers`.